dags/hindawi/hindawi_file_processing.py

```diff
@@ -27,7 +27,7 @@ def hindawi_validate_record(enriched_file):
     validate(enriched_file, schema)
 
 
-@dag(start_date=airflow.utils.dates.days_ago(0))
+@dag(schedule_interval=None, start_date=airflow.utils.dates.days_ago(0))
```

Comment:
do we need to keep the `start_date`?

Reply:
yes, we need it, otherwise, we need to put the start_date on every task:
https://airflow.apache.org/docs/apache-airflow/1.10.4/_modules/airflow/models/dag.html